src/main/java/com/skywalx/simpleplayerauthentication/config/MessageConfiguration.java

```diff
@@ -1,14 +1,18 @@
 package com.skywalx.simpleplayerauthentication.config;
 
+import me.clip.placeholderapi.PlaceholderAPI;
+import org.bukkit.Bukkit;
```

Comment:
Remove unused import

Reply:
Thanks, enabled the optimize import check when committing again! 
Seems like it wasn't on this OS install :D